test/unit/workers/worker_base_test.rb

```diff
@@ -30,26 +30,10 @@ def perform(*_args)
     example_arg = stub("example arg")
     WorkerBase.expects(:client_push).with(
       'class' => WorkerBaseTest::MyWorker,
-      'args' => [example_arg, { authenticated_user: nil, request_id: nil }],
+      'args' => [example_arg],
       'queue' => 'my-test-queue'
     )
     MyWorker.perform_async_in_queue(nil, example_arg)
   end
-
-  test ".perform_async_in_queue populates header variables" do
```

Comment:
The WorkerBase test on headers was removed because the WorkerBase is not adding headers anymore. The govuk_sidekiq gem does it when calling the `perform` method.